gradio/flagging.py

```diff
@@ -163,7 +163,7 @@ def replace_flag_at_index(file_content):
         if self.encryption_key:
             output = io.StringIO()
             if not is_new:
-                with open(log_filepath, "rb") as csvfile:
+                with open(log_filepath, "rb", encoding="utf-8") as csvfile:
```

Comment:
Separate problem (on Windows) was that the special characters in Turkish were not being encoded correctly